### ISSUE TITLE
Message when failing to install a package.

### ISSUE THREAD

**Issue by Splike** [atom/atom#7085]
When receiving the "failed to install package" message, this message is not displayed when clicking install on the actual package page. Instead, this message shows up in the view, where you actually search for other packages.

Example:
I am on the package page of "lint-clang". The button started doing its installing animation and then stopped, returning the the normal "Install" button. I tried clicking it a few more times, but it would not install.
I go back to the view, where I searched for package, and I see an error, saying the package failed to install. "Failed to install linter-clang because Git was not found.".

In conclusion, error messages for installing packages, should pop-up on the package page.

**Comment by mnquintana** [comment]
Would you mind moving this to https://github.com/atom/settings-view?

**Comment by mnquintana** [comment]
This issue was moved to atom/settings-view#534
